docs/book/v1/intro.md

```diff
@@ -0,0 +1,108 @@
+# zend-expressive-authentication-basic
```

Comment:
The content of this headline should be "Introduction", because:

- the [page](https://github.com/zendframework/zend-expressive-authentication-basic/blob/149b70db37689d0bc2e89ac36ed9d46ffb614a25/mkdocs.yml#L5) is named with "Introduction"
- the [homepage](https://docs.zendframework.com/zend-expressive-authentication-basic/) already has this headline